src/PKSim.Presentation/Presenters/Compounds/CompoundProcessesPresenter.cs

```diff
@@ -386,17 +390,17 @@ public void RemoveProcess(CompoundProcess compoundProcess)
 
       public void AddSystemicProcess(IEnumerable<SystemicProcessType> systemicProcessType)
       {
-         AddCommand(_compoundProcessTask.CreateSystemicProcessFor(_compound, systemicProcessType));
+         AddCommand(_compoundProcessPresentationTask.CreateSystemicProcessFor(_compound, systemicProcessType));
```

Comment:
There might be a better way to split the tasks that requires user input from the other. But this is good enoufh for now